src/sentry/ingest/consumer/processors.py

```diff
@@ -188,12 +188,8 @@ def process_event(
             )
         elif data.get("type") == "feedback":
             if features.has("organizations:user-feedback-ingest", project.organization, actor=None):
-                save_event_feedback.delay(
-                    cache_key=None,  # no need to cache as volume is low
-                    data=data,
-                    start_time=start_time,
-                    event_id=event_id,
-                    project_id=project_id,
+                create_feedback_issue(
```

Comment:
Can we make this a feature flag first?

Reply:
@anonrig `save_event_feedback` (the celery task) is just a wrapper for `create_feedback_issue`, so I don't think this is necessary